### PR TITLE
Develop book/inventory objects

### LINKED ISSUE #4: Develop Book/Inventory Objects
- Create book object
- Create inventory object
- Create book repo
- Create inventory repo

### PR DESCRIPTION
**Pull request by j-ctrl7**

### CODE REVIEW

File: src/main/java/project/Inventory.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    @Id
+    @GeneratedValue
+    private Long id;
+    @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+    private List<Book> books;
+
+    public Inventory(){
+        this.books = new LinkedList<>();
+    }
+
```
Comment (chase-scott) [inline src/main/java/project/Inventory.java]: Will need to add methods for adding, deleting, and editing books.

File: src/main/java/project/Book.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
+package project;
+import java.util.*;
+import jakarta.persistence.*;
+
+@Entity
+public class Book {
+    @Id
+    private Long isbn;
```
Comment (chase-scott) [inline src/main/java/project/Book.java]: maybe make this all capitalized for style

**Comment by chase-scott** [comment]
needs to be in the proper file structure

**Review by Favourolotu — COMMENTED** [review]
Need getters and setters for the books in the inventory model

**Comment by chase-scott** [comment]
Looks good now 👍

**Review by Favourolotu — APPROVED** [review]
Looks good now

**Pull request merged**